aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -210,20 +217,20 @@ struct Vec256 {
     return ret;
   }
   template <typename other_t_abs = T,
-            typename std::enable_if<!std::is_floating_point<other_t_abs>::value && !c10::is_complex_t<other_t_abs>::value, int>::type = 0>
+            typename std::enable_if<!is_floating_point<other_t_abs>::value && !c10::is_complex_t<other_t_abs>::value, int>::type = 0>
   Vec256<T> abs() const {
     // other_t_abs is for SFINAE and clarity. Make sure it is not changed.
     static_assert(std::is_same<other_t_abs, T>::value, "other_t_abs must be T");
     return map([](T x) -> T { return x < static_cast<T>(0) ? -x : x; });
   }
   template <typename float_t_abs = T,
-            typename std::enable_if<std::is_floating_point<float_t_abs>::value, int>::type = 0>
+            typename std::enable_if<is_floating_point<float_t_abs>::value, int>::type = 0>
   Vec256<T> abs() const {
     // float_t_abs is for SFINAE and clarity. Make sure it is not changed.
     static_assert(std::is_same<float_t_abs, T>::value, "float_t_abs must be T");
     // Specifically deal with floating-point because the generic code above won't handle -0.0 (which should result in
     // 0.0) properly.
-    return map(std::abs);
```

Comment:
huh, what's going on here?

Reply:
There is no `std::abs(at::Half)` so this allows implicit conversions to float.